apps/mobile/src/features/browser/approver-sheet/send-transfer.tsx

```diff
@@ -0,0 +1,118 @@
+import { useMemo } from 'react';
+
+import { PsbtSigner } from '@/features/psbt-signer/psbt-signer';
+import { createCoinSelectionUtxos } from '@/features/send/utils';
+import { useAverageBitcoinFeeRates } from '@/queries/fees/fee-estimates.hooks';
+import { useAccountUtxos } from '@/queries/utxos/utxos.query';
+import { App } from '@/store/apps/utils';
+import { useBitcoinAccounts } from '@/store/keychains/bitcoin/bitcoin-keychains.read';
+import { destructAccountIdentifier } from '@/store/utils';
+import { bytesToHex } from '@noble/hashes/utils';
+import BigNumber from 'bignumber.js';
+
+import {
+  createBitcoinAddress,
+  generateBitcoinUnsignedTransactionNativeSegwit,
+  getBtcSignerLibNetworkConfigByMode,
+  payerToBip32Derivation,
+} from '@leather.io/bitcoin';
+import { AverageBitcoinFeeRates, bitcoinNetworkModesSchema } from '@leather.io/models';
+import { RpcRequest, RpcResponse, createRpcSuccessResponse, sendTransfer } from '@leather.io/rpc';
+import { UtxoTotals } from '@leather.io/services';
+import { createMoneyFromDecimal } from '@leather.io/utils';
+
+interface SendTransferApproverProps {
+  request: RpcRequest<typeof sendTransfer>;
+  sendResult(result: RpcResponse<typeof sendTransfer>): void;
+  app: App;
+  closeApprover(): void;
+  accountId: string;
+}
+
+export function SendTransferApprover(props: SendTransferApproverProps) {
+  const { data: feeRates } = useAverageBitcoinFeeRates();
+  const { fingerprint, accountIndex } = destructAccountIdentifier(props.accountId);
+  const utxos = useAccountUtxos(fingerprint, accountIndex);
+
+  if (!utxos.value) return null;
```

Comment:
If `!utxos.value` or `!feeRates` then we return `null` so do we show an empty section / let them retry? 🤔